src/codec_aom.c

```diff
@@ -876,6 +876,11 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                 return AVIF_RESULT_UNKNOWN_ERROR;
             }
         }
+
+        if (image->depth == 12) {
```

Comment:
I wonder if we should only turn this off when we are encoding animations (`image->depth == 12 && cfg->g_limit != 1`).

Reply:
We can start with that.

Reply:
Plese ignore this suggestion. This needs more investigation.

Reply:
Sorry, I think there was a race. If you're OK with it as is we can let fuzzing continue, otherwise I'll undo the check in a follow up.

Reply:
This check satisfied the fuzzing cases I tested, but I posted https://github.com/AOMediaCodec/libavif/pull/2702 if you'd like it.